src/components/icons/LeftArrowIcon.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+interface AddIconProps extends React.SVGProps<SVGSVGElement> {}
+
+const DownArrowIcon: React.FC<AddIconProps> = () => {
```

Comment:
FIle name is leftArrowIcon but exported component is downArrowIcon?

Reply:
Sorry, this file is not required. Deleting it